lib/pages/home_page.dart

```diff
@@ -0,0 +1,166 @@
+                                    height * 0.05,
+                                  ),
+                                  Text(
+                                    style: Theme.of(context).textTheme.
+                                    headlineMedium,
+                                    textAlign: TextAlign.center,
+                                    'Home Page',
+                                  ),
+                                  Container(
+                                    width: 400,
+                                    height: 600,
+                                    decoration: BoxDecoration(
+                                      borderRadius: BorderRadius.circular(40),
+                                    ),
+                                    child: const Padding(
+                                      padding: EdgeInsets.all(20),
+                                      child: Column(
+                                        children: [
+                                          SizedBox(
+                                            height: 20,
+                                          ),
+                                          Text(
+                                            'This is home page. Here will be '
+                                            'all information about your budget,'
+                                            ' your incomes and outcomes. '
+                                            'Bellow is the pie-chart to see '
+                                            'your every day progress:',
+                                            style: TextStyle(fontSize: 16),
+                                          ),
+                                          SizedBox(
+                                            height: 20,
+                                          ),
+                                          Icon(
+                                            Icons.circle_outlined,
+                                            size: 230,
+                                          ),
+                                          SizedBox(
+                                            height: 30,
+                                          ),
+                                          Column(
+                                            children: [
+                                              CurrentDay(),
+                                            ],
+                                          ),
+                                        ],
+                                      ),
+                                    ),
+                                  ),
+                                ],
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
```

Comment:
Tooooooo much trailing coma)
Please divide as a separate widget and reuse

Reply:
Fixed, thanks